src/views/Portal/components/Background/index.scss

```diff
@@ -1,7 +1,33 @@
 .application-background {
-    &__box {
-        background: rgba(255, 251, 241, 0.83);
-        box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.41);
-        border-radius: 18px;
+    position: relative;
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    min-height: 100vh;
+    height: fit-content;
+    background: #FFF2D9;
+    z-index: 0;
+
+    &__hills {
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        z-index: -1;
     }
-}
\ No newline at end of file
+
+    &__waves {
+        position: absolute;
+        // 1440 x 385 => ~26% width = top offset of wave
+        top: 24vw;
+        left: 0;
+        bottom: 0;
+        width: 100%;
+        height: calc(100% - 24vw);
+        z-index: -1;
+    }
+
+    &__context {
+        z-index: 1;
+    }
```

Comment:
Should we apply styling here so that the children is starting in roughly the same vertical position? We could also just handle that from child container as well.

Reply:
Yeah, I can add that! You could also merge this in first if this PR is blocking anyone, since that change will probably happen in the weekend.

Reply:
Sounds good! If we can just clear the merge conflict, we can merge this in and move forward from there!

Reply:
On second thought we should probably handle this in the child? Since what we really want to do is to keep the box in the same position, but the `Section X/X` thing is also a child.

<img src="https://user-images.githubusercontent.com/20177171/140560722-6811a5b3-c163-4352-a3a2-e910d12c3fb9.png" width="350px" />

Reply:
That makes sense. There is still one merged conflict, most likely from my branch that I merged but we should be good.